sass/elements/image/_image.scss

```diff
@@ -3,7 +3,18 @@
   Generic image classes
   These classes are set from silverstripe html editor
 */
+$default-element-img-properties:(
+    borderRadius: initial,
+    boxShadow: none
+);
+// If $element-img-properties is set, lets merge into our defaults
+$element-img-properties: $default-element-img-properties !default;
+@if($element-img-properties){
+  $element-img-properties: recursive-map-merge($default-element-img-properties, $element-img-properties);
+}
 img{
+  border-radius: getThemeProperty(borderRadius, $element-img-properties);
+  box-shadow: getThemeProperty(boxShadow, $element-img-properties);
```

Comment:
We should look at limiting these sorts of styles as this effects every img tag on the site, this could effect logos, etc...

Maybe it should only be applied to "content" related images?